packages/metrics/src/index.ts

```diff
@@ -1,4 +1,4 @@
-import Raven from 'raven-js';
+import { init as initSentry, configureScope, BrowserOptions } from '@sentry/browser';
```

Comment:
Can we export all of sentry here?  Otherwise apps can't really take advantage of most of the setup if they want to use any additional options.  For example we have an on click event that logs breadcrumbs.  There are other context events that sentry exposes as well.

Reply:
I can just make `@sentry/browser` a peer dep. There's a lot of stuff in this package, so pick and choosing what to re-export seems overkill.

Reply:
👍 I was thinking `export * as Sentry from '@sentry/browser`, but peer dep works too.

Reply:
Actually, exporting it under a namespace isn't a bad idea either.